### ISSUE TITLE
Combination of several listed Realtek Audio Console issues

### ISSUE THREAD

**Issue by Kuiriel** [pal1000/Realtek-UAD-generic#131]
TL;DR: 
1. Trying to install UAD Generic on ASUS z790 Apex, Windows 11 64 bit, so I can use RTKConfig to enable simultaneous front and back audio streams.
2. No obvious error messages besides mentioning rtkauduservice64.exe doesn't exist right at the beginning.
3. RAC can't connect to RPC Service.
4. After installation, Device Manager doesn't list Realtek devices under Software Components and instead lists 'Generic Software Component'.
5. C:\windows\system32\rtkauduservice64.exe does not get installed.
6. Manually forcing rtkauduservice64.exe -background from C:\Windows\System32\DriverStore\FileRepository\realtekservice.inf_amd64_18daf79b2c8abbf6 results in RAC switching to black screen and error "realtek audio console doesn't support for this machine."
7. Re-installing official drivers still does not give access back to RPC Service, which now appears to be gone forever. Looks like it's a listed bug in the Windows Store reviews as well. The service is still not listed in regedit or startup apps.

Process undertaken thus far:
Windows 11 64 bit, latest version as of today, Windows 11 freshly install from a month ago.

Downloaded Unofficial-Realtek-UAD-generic-6.0.9464.1.zip

Ran the UAD driver, didn't automatically install RAC which I for some reason expected.

Used offline installer for RAC. Same issue where it has a loading circle beneath 'audio console' for ages until eventually it states "RAC can't connect to RPC service".

I tried opening Microsoft Store and it didn't work so I ran wsreset

I can't find Realtek Audio Control in the MS Store app, can see a bunch of other junk though. If I click the link on the web page to go to 'get in store app' it doesn't show up in the Store app. I'm not using a Live account though, just a local one.

I tried turning off real time virus detection in Windows Defender and reinstalled from the offline package.

Same issue. RAC can't connect to RPC service. Uninstalled RAC again.

Went into Privacy & Security > For Developers > Developer Mode flicked to [ON] to install apps from any source, as no 'sideload' option was visible.

Reinstalled RAC. 

Same issue. RAC can't connect to RPC service.

For some reason I can't see any Realtek services under Service though.

C:\windows\system32\rtkauduservice64.exe does not exist, nor is it listed under Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Run.

Instead I can find it under 

C:\Windows\System32\DriverStore\FileRepository\realtekservice.inf_amd64_18daf79b2c8abbf6

I don't think that's the right spot though.

I've gone through uninstalling and reinstalling again, still no luck. I still don't get a Realtek Service up and running in the registry or services. The only realtek driver I have not removed is the bluetooth rtkfilter.inf file. Everything else realtek is being added by the UAD Generic installer.

![DriverStoreExplorerAfterUADInstall](https://user-images.githubusercontent.com/24553989/213116713-b70645db-f6ae-4cc4-a33c-78b0278ce63f.png)

I tried right clicking and installing directly via C:\Drivers\Realtek-UAD-generic\Win64\Realtek\RealtekService_591 - realtekservice.inf but still nothing. Going to reboot.

No luck. I went to C:\Windows\System32\cmd.exe and tried "RtkAudUService64.exe -background"

I tried running Realtek Audio Console. Now there is a black screen with the error "realtek audio console doesn't support for this machine."

I tried running disabling internet again, running DDU, removing everything, rebooting, checking again with DriverStoreExplorer to remove another, running UAD Generic installer again - there are no errors in the log that I can see except for right at the beginning when it says there's no realtek service to stop. 

And yet it is still not installing a Realtek Service for the RAC to function.

Starting to think the codecs or something in my z790 Apex motherboard isn't supported by this lovely program.

For some reason under Device Manager > Software Devices > I have listed Generic Software Component instead of Realtek bits and pieces. Under Sound, video and game controllers I have Realtek USB2.0 Audio listed though, which is how my audio comes through... even after using DDU and DriverStoreExplorer? So why isn't usbaudio2.inf showing up DriverStoreExplorer for me to remove it? Removed it again, but after reboot Windows just brought it back.

If I use the drivers from ASUS which are Realtek USB Audio Driver V6.3.9600.2342 For Windows 10/11 64-bit then I am able to use Windows Store to find, download and install Realtek Audio Control. This changes the listing under Sound, video and game controllers to Realtek USB Audio instead of USB2.0 Audio, go figure.

The Audio Console is still borked however and can't connect to the service. So I've gone from functioning access to a RAC to having no access. Blargh.

**Comment by pal1000** [comment]
Realtek-UAD-generic doesn't support  Realtek USB Audio devices.

**Comment by Kuiriel** [comment]
Sorry, I'm confused. I have the z790 Apex. I have a front and back audio port that looks analogue to me, regular little stereo jacks. I'm not actually using any 'usb' ports for my audio. Does this mean that my entire inside expensive motherboard is running on some sort of analogue to USB conversion setup?

Is that what it means when it says Realtek USB Audio on Device Manager?

Still not sure how I've lost my realtek audio console in the entire process when I'm sure I had it before, must be some issue with the official motherboard drivers being installed now.

**Comment by Kuiriel** [comment]
Thanks again. I will continue to pursue solutions in other places, but appreciate the joy your work brought me over previous years.
